src/App.js

```diff
@@ -25,13 +27,14 @@ function App() {
         <Route path={LOGIN_URL} component={LoginPage} />
         <PrivateRoute path={ADD_ARTICLE_URL} component={AddArticle} />
         <PrivateRoute
-          path={ARTICLE_DETAILS_URL + "/:id"}
-          component={ArticleDetails}
+          path={EDIT_ARTICLE_URL + "/:id"}
```

Comment:
it is better to keep the whole URL in the `EDIT_ARTICLE_URL` constant, i.e: keep the `/:id` in the constant itself

Reply:
Done